homework.md

```diff
@@ -92,13 +95,13 @@ A place for us to store and share our code.
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+Send your code to Github (bring the changes back to GitHub)
```

Comment:
Hi Hozan,
nice work so far, you need to think a bit more for this one, you are confusing commit with push.

Reply:
Hi iTrip23, 

I think the answer might be right beacuse we when we commit, we send changes back to GitHub.

And push is used to send the these commits to the remote repo.

maybe @ChrisOwen101 can correct me if I am wrong.

Reply:
@Hozan94 Github is the remote repository so `push` is the command to send changes to the Github.

So, what do you think `commit` does then?

Reply:
@louisechow then `commit` send changes to the local repository (the repo that is stored on your local computer ? ).

Also if that is the case then how come you have the option to make `commit` on GitHub Web, even though you have not cloned the repo?